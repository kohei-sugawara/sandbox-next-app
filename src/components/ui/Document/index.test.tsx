import { render } from '@testing-library/react';
import { Document } from '.';

describe('ui/Document', () => {
  it('renders correctly', () => {
    const { container } = render(<Document />);
    expect(container).toMatchSnapshot();
  });
});
