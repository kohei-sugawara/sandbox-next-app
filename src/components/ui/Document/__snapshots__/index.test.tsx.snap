// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui/Document renders correctly 1`] = `
<div>
  <h1>
    Document
  </h1>
</div>
`;
